testutils/base.go

```diff
@@ -24,22 +24,20 @@ import (
 	"github.com/cockroachdb/cockroach/security"
 )
 
-// NewTestBaseContext creates a base context for testing.
+// NewRootTestBaseContext creates a base context for testing.
 // This uses embedded certs and the "root" user (default client user).
-// The certs file loader is overriden in individual main_test files.
-// This is meant to be used by external clients (as opposed to nodes).
-func NewTestBaseContext() *base.Context {
+// The "root" user has client certificates only.
+func NewRootTestBaseContext() *base.Context {
```

Comment:
Rather than having separate `New{Root,Node}TestBaseContext` methods, I wonder if this would be cleaner and clearer to explicitly pass the user. The advantage of separate methods is that they can start to diverge in more than just the user associated with the context. But if you don't anticipate such divergence a single method would make it clearer what is varying between the contexts.